server-environment.md

```diff
@@ -10,13 +10,13 @@ The web server is piece of software that accepts user web requests and serves th
 The two most popular ones that are recommended are:
 
 * [Apache HTTPD](https://httpd.apache.org/) 2.4
-* [nginx](https://nginx.org/) 1.20 / 1.19
+* [nginx](https://nginx.org/) 1.21 / 1.20
```

Comment:
1.21 isn't stable, it's "mainline" aka fast moving version.
1.22 is the current stable version, just released yesterday. 1.20 was stable before this.